include/swift/AST/Decl.h

```diff
@@ -1981,10 +1985,12 @@ class PatternBindingEntry {
     InitExpr.EqualLoc = equalLoc;
   }
 
-  /// Retrieve the initializer as it was written in the source.
-  Expr *getInitAsWritten() const {
-    return InitContextAndIsText.getInt() ? nullptr : InitExpr.Node;
-  }
+  /// Retrieve the initializer after the =, if any, as it was written in the
+  /// source.
+  Expr *getOrigInit() const;
+  
+  /// Set the initializer after the = as it was written in the source.
+  void setOrigInit(Expr*);
```

Comment:
I think even in function declarations we prefer to name arguments, also we put space before the *

Reply:
In this case, though, the argument name adds nothing to clarity, AFAICT. That's why I thought the code was better without it. When the argument is a bool, or there are more than one argument, esp. of the same type, then I agree that the names are  important. (Emerson said it best...)